sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_first_seen_v1/metadata.yaml

```
@@ -3,15 +3,30 @@ friendly_name: FxA Users Services First Seen
 description: Usage aggregations describing when each FxA user was first seen
 owners:
   - dthorn@mozilla.com
+  - kignasiak@mozilla.com
```

there can only be one owner (it's not an array field in airflow). IDK why it's an array field here, but for the scope of this PR, we'll just remove me from the list for this task:

```suggestion
owners:
  - kignasiak@mozilla.com
```